### PR TITLE
Adds error if pokemons' points sum more than 600

### PR DESCRIPTION
**Pull request by jaenia**
[As a Trainer I see form errors in case the PKNs I select sum more than 600 points](https://trello.com/c/3lauyxqC/17-as-a-trainer-i-see-form-errors-in-case-the-pkns-i-select-sum-more-than-600-points)

### Description
This PR adds error if points of selected Pokemons sum more than 600.

### How to test
- [ ] Click to add a battle
- [ ] Select your opponent
- [ ] Fill the Pokemons with the following ids: 493 (Arceus), 646 (Kyurem), and 150 (Mewtwo).
- [ ] Check that you cannot create a battle if the points sum more than 600

### CODE REVIEW

File: backend/battles/forms.py [hunk 1]
```diff
@@ -41,32 +41,51 @@ def __init__(self, *args, **kwargs):
...
     def clean(self):
         cleaned_data = super().clean()
         cleaned_data["creator"] = self.fields["creator"].initial
 
+        creator_pokemon_1_input = cleaned_data.get("creator_pokemon_1_input")
+        creator_pokemon_2_input = cleaned_data.get("creator_pokemon_2_input")
+        creator_pokemon_3_input = cleaned_data.get("creator_pokemon_3_input")
+
+        pokemon_points_sum = 0
+        if self.is_valid():
```
Comment (filipeximenes) [inline backend/battles/forms.py]: You shouldn't be calling `is_valid` inside the `clean` method

File: backend/pokemons/helpers.py [hunk 2]
```diff
@@ -18,3 +18,21 @@ def save_pokemon(poke_id):
     )
 
     return pokemon
+
+
+def get_pokemons_points_sum(poke_ids):
```
Comment (filipeximenes) [inline backend/pokemons/helpers.py:24]: Cool!

File: backend/pokemons/helpers.py [hunk 3]
```diff
@@ -18,3 +18,21 @@ def save_pokemon(poke_id):
...
+    points_sum = 0
+    for poke_id in poke_ids:
+        pokemon = Pokemon.objects.filter(poke_id=poke_id).first()
```
Comment (filipeximenes) [inline backend/pokemons/helpers.py:27]: Cool!

**Review by filipeximenes — CHANGES_REQUESTED** [review]
Good job! One minor comment

File: backend/battles/forms.py [hunk 4]
```diff
@@ -41,32 +41,53 @@ def __init__(self, *args, **kwargs):
...
+        creator_pokemon_1_input = cleaned_data.get("creator_pokemon_1_input")
+        creator_pokemon_2_input = cleaned_data.get("creator_pokemon_2_input")
+        creator_pokemon_3_input = cleaned_data.get("creator_pokemon_3_input")
+
+        pokemon_points_sum = 0
+        try:
+            pokemon_points_sum = get_pokemons_points_sum(
+                [creator_pokemon_1_input, creator_pokemon_2_input, creator_pokemon_3_input]
+            )
+        except:  # noqa #nosec
```
Comment (filipeximenes) [inline backend/battles/forms.py]: you should always specify the exception. There's only an specific error we want to ignore. By ignoring all exceptions we are allowing for errors that are not expected to pass by silently.

File: backend/pokemons/helpers.py [hunk 5]
```diff
@@ -18,3 +19,23 @@ def save_pokemon(poke_id):
...
+
+        if pokemon:
+            points_sum += pokemon.attack + pokemon.defense + pokemon.hit_points
+        else:
+            if not pokemon_exists(poke_id):
+                raise PokemonNotFound(poke_id)
```
Comment (filipeximenes) [inline backend/pokemons/helpers.py:33]: Where's the test for this? Remember to use TDD!

**Review by filipeximenes — CHANGES_REQUESTED** [review]

File: backend/battles/forms.py (hunk 4 above)
Comment (jaenia) [reply backend/battles/forms.py]: Got it. Thank you!

File: backend/pokemons/helpers.py (hunk 5 above)
Comment (jaenia) [reply backend/pokemons/helpers.py:33]: Haha, thanks for the catch and for remember me about TDD!

File: backend/battles/forms.py [hunk 6]
```diff
@@ -41,32 +42,53 @@ def __init__(self, *args, **kwargs):
...
+        creator_pokemon_1_input = cleaned_data.get("creator_pokemon_1_input")
+        creator_pokemon_2_input = cleaned_data.get("creator_pokemon_2_input")
+        creator_pokemon_3_input = cleaned_data.get("creator_pokemon_3_input")
+
+        pokemon_points_sum = 0
+        try:
+            pokemon_points_sum = get_pokemons_points_sum(
+                [creator_pokemon_1_input, creator_pokemon_2_input, creator_pokemon_3_input]
+            )
+        except PokemonNotFound:  # noqa #nosec
```
Comment (filipeximenes) [inline backend/battles/forms.py]: You should be able to remove `# noqa #nosec` now

**Review by filipeximenes — APPROVED** [review]
😃

**Pull request merged**